src/components/player/cue-bar.tsx

```diff
@@ -79,22 +79,23 @@ const NoteCue: React.FC<any> = ({
   className,
   actions,
   player,
-  onClickCue,
+  onClickCue, // we might want to do other stuff with this, but player state covers the same base
 }) => {
   const setVisible = useCue(cue, actions)
-  const [persist, setPersist] = React.useState(false)
-  const show = () => setVisible(true)
-  const hide = () => setVisible(false)
+
   const open = () => {
-    show()
-    setPersist(true)
-    onClickCue()
+    setVisible(true)
+    // if we seek to the correct time, the note is displayed
+    actions.seek(cue.startTime)
```

Comment:
there was a difference here with show/open, but I removed that for now. IMO a cue is active or not so I'm relying on that mostly

also, instead of tracking clicks or other approach I "seek" the player to the appropriate time (cue start) and it "just works"

Reply:
this sounds good in theory, however I'm getting weird results: <img src="https://p-ZmFjNlQ.b3.n0.cdn.getcloudapp.com/items/04u2NDRr/acf9db2a-27c0-450c-ae56-70d61d907e2f.gif?v=2664226f610363fd6effe4bd5abb753a" />

plus sidebar stopped scrolling to active note?